apps/laboratory/package.json

```diff
@@ -13,7 +13,7 @@
     "playwright:test:email": "playwright test --grep 'email.spec.ts'",
     "playwright:test:siwe": "playwright test --grep siwe.spec.ts",
     "playwright:test:sa": "playwright test --grep smart-account.spec.ts",
-    "playwright:test:canary": "playwright test --retries=0 --grep canary.spec.ts --project='Desktop Chrome/wagmi'",
+    "playwright:test:canary": "playwright test --retries=0 --grep canary.spec.ts --project='Desktop Firefox/ethers'",
```

Comment:
What are you switching this?

Reply:
This was for local testing purpose, should have been missed. Will revert back.

Reply:
Btw, why it's testing only on wagmi?

Reply:
Because the canary is supposed to test if the system is generally working together. E.g. is the relay down, and what is the overall latency of the system over-time. Chromium/Wagmi is the sort of best-case configuration/recommendation and is least likely to fail from test flakes